src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -55,21 +55,27 @@ that a given scan was collected with the intended coil elements selected
 |--------------------------------|-----------------------|---------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| NumberShots                    | RECOMMENDED           | [number][]    | The number of RF excitations need to reconstruct a slice or volume. Please mind that this is not the same as Echo Train Length which denotes the number of lines of k-space collected after an excitation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| NumberShots                    | RECOMMENDED           | [number][] or [array][] of [numbers]    | The number of RF excitations needed to reconstruct a slice or volume (may be referred as partition). Please mind that this is not the same as Echo Train Length which denotes the number of k-space lines collected after excitation in a multi-echo readout.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
> (may be referred as partition)

I think this needs a "to" (may be referred to as partition).

Reply:
2fd6edc